jib-gradle-plugin/src/integration-test/resources/projects/simple/complex-build.gradle

```diff
@@ -34,7 +34,7 @@ jib {
         args = ['An argument.']
         mainClass = 'com.test.HelloWorld'
         jvmFlags = ['-Xms512m', '-Xdebug']
-        environment = [var1:'value1', var2:'value2']
+        environment = [env1:'value1', env2:'value2']
```

Comment:
Oh I was referring to `value1` and `value2` since it would make the `assertEquals` be something like `\nenv1value\nenv2value\n` rather than `\nvalue1\nvalue2\n` where `value1` and `value2` are not immediately clear where they should have come from.